ts/themes/oceanDark.ts

```diff
@@ -51,7 +51,7 @@ export const oceanDark: ThemeColorVariables = {
   '--button-outline-border-hover-color': 'var(--text-primary-color)',
   '--button-outline-disabled-color': 'var(--disabled-color)',
 
-  '--button-solid-background-color': 'var(--background-secondary-color)',
+  '--button-solid-background-color': THEMES.OCEAN_DARK.COLOR1,
```

Comment:
This change breaks the solid button. When we don't set the color to something like Danger.

Before:
<img width="1341" alt="Screenshot 2024-12-04 at 10 55 44" src="https://github.com/user-attachments/assets/28444505-d2de-44c7-ba8e-68bace33af54">

After:
<img width="1338" alt="Screenshot 2024-12-04 at 10 55 38" src="https://github.com/user-attachments/assets/1fc8e0f9-60e7-4e4f-8766-440590e3f8f0">


Also the colors for solid buttons has changed in the design system and I updated them in the network page. You could take the colors from my PR if you like